_stbt/ocr.py

```diff
@@ -363,6 +391,57 @@ def match_text(text, frame=None, region=Region.ALL,
     return result
 
 
+# Python 2.7 & 3.6 have `re._pattern_type` but that will be removed in Python
+# 3.7 where they introduce `re.Pattern`.
+PatternType = type(re.compile(""))
+
+
+def apply_ocr_corrections(text, corrections=None):
+    """Applies the same corrections as `stbt.ocr`'s ``corrections`` parameter.
+
+    This is also available as a separate function, so that you can use it to
+    post-process old test artifacts using new corrections. See also
+    `stbt.set_global_ocr_corrections`.
+    """
+    if corrections:
```

Comment:
I think this should be `corrections is not None`.  That way you can pass `corrections={}` to disable corrections.